schemas/telemetry/error-schema.json

```diff
@@ -9,31 +9,72 @@
       "$ref": "_common-schema.json"
     },
     {
-      "required": [
-        "status",
-        "message"
-      ],
+      "required": ["telemetry"],
       "properties": {
-        "status": {
-          "type": "string",
-          "description": "Level/severity of the log",
-          "const": "error"
-        },
-        "message": {
-          "type": "string",
-          "description": "Body of the log"
-        },
-        "error": {
+        "telemetry": {
           "type": "object",
-          "description": "Error properties",
+          "description": "The telemetry information as it should be indexed",
+          "required": ["status", "message"],
           "properties": {
-            "stack": {
+            "status": {
               "type": "string",
-              "description": "The stack trace or the complementary information about the error"
+              "description": "Level/severity of the log",
+              "const": "error"
             },
-            "kind": {
+            "message": {
               "type": "string",
-              "description": "The error type or kind (or code in some cases)"
+              "description": "Body of the log"
+            },
+            "error": {
+              "type": "object",
+              "description": "Error properties",
+              "properties": {
+                "stack": {
+                  "type": "string",
+                  "description": "The stack trace or the complementary information about the error"
+                },
+                "kind": {
+                  "type": "string",
+                  "description": "The error type or kind (or code in some cases)"
+                },
+                "application": {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Are those three fields expected to be nested in `telemetry.error`? Shouldn't it be in `telemetry` directly, like for debug telemetry events?

Reply:
My bad that's correct